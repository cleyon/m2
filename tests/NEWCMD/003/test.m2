@newcmd guilty{who}
@for i 1 4
That's guilty, @who@, guilty!
@next i
@endcmd
@guilty{Nixon}
@guilty{Trump}
