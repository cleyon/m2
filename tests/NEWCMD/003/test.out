That's guilty, Nixon, guilty!
That's guilty, Nixon, guilty!
That's guilty, Nixon, guilty!
That's guilty, Nixon, guilty!
That's guilty, Trump, guilty!
That's guilty, Trump, guilty!
That's guilty, Trump, guilty!
That's guilty, Trump, guilty!
